sdk/rust/quickstart.mdx

```diff
@@ -11,41 +11,49 @@ In this Rust quickstart we will learn how to:
 - Execute a query using SQL
 
 <Steps>
-  <Step title="Install libsql-client">
+  <Step title="Install libsql crate">
```

Comment:
@xinnks let's make sure we include the new Snippet for obtaining database credentials. You might need to rebase with `main`.